qiskit/providers/models/backendconfiguration.py

```diff
@@ -400,16 +411,66 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, qubits: Iterable[int] = None,
+                channel: int = None) -> List[ControlChannel]:
         """
         Return the secondary drive channel for the given qubit -- typically utilized for
         controlling multiqubit interactions. This channel is derived from other channels.
 
+        Args:
+            qubits: Tuple or list of qubits of the form `(control_qubit, target_qubit)`.
+            channel: Deprecated.
+
+        Raises:
+            BackendConfigurationError: If the qubit is not a part of the system or if
+                given ``qubits`` is not found.
+
         Returns:
-            Qubit control channel.
+            List of control channel.
         """
         # TODO: Determine this from the hamiltonian.
-        return ControlChannel(channel)
+        if channel is not None:
+            warnings.warn('control(channel: int) is deprecated. Use '
+                          'control(qubits: List or Tuple [control_qubit, target_qubit]) instead.',
+                          DeprecationWarning)
+            return ControlChannel(channel) # TODO: Fix this logic
```

Comment:
need to fix this logic `return ControlChannel(channel)` since expected value should be a list/tuple